### PR TITLE
Use direct imports of PF4 components

### PR DESCRIPTION
**Pull request by Hyperkid123**
This changes how we are importing PF components to improve the build size of JS chunks

I have also locked the PF version to `3.141.4` due to a bug withing the pagination component which causes an additional 0.5MB of react-tokens assets to be bundled with the component and it was not fixed yet. (issue here: https://github.com/patternfly/patternfly-react/issues/4184). We could ever that I have not found any critical piece of code in the newer version that would affect these bundles.

I have tried and use babel transform plugin to do this import transformation while building, but parcel does not support `babel.config.js` files, because PF requires some fs searching to actually find correct files and that cannot be done in `.babelrd` or `babel.json` :(

There is an eslint plugin from PF that checks the imports, but as I tried to add it to this project it completely messed up the linter and I did not want to change every single file in order to fix this issue (plugin: https://github.com/patternfly/patternfly-react/tree/master/packages/eslint-plugin-patternfly-react).

### Before (only including the JS and CSS files because its relevant don't care about map files and images)
```bash
build/js/chrome.js                                           ⚠️  4.59 MB    52.46s
build/js/frontend-components-inventory.e3560e4d.js            892.99 KB    47.50s
build/js/chrome.css                                           439.84 KB    28.85s
build/js/frontend-components-remediations.a65e56a1.js         101.55 KB    23.39s
build/js/App.eeab7546.js                                       44.59 KB    26.24s
build/js/inventoryStyles.dee582d6.js                             1.4 KB     5.64s
build/js/remediationsStyles.83d6d9d2.js                         1.35 KB     3.33s
build/js/remediationsStyles.a24650e6.css                        1.22 KB     1.75s
build/js/App.30fd3cd2.css                                         942 B    13.39s

```

### After
```bash
build/js/chrome.js                                            ⚠️  1.07 MB    30.50s
build/js/frontend-components-inventory.b5cf168a.js            ⚠️  1.01 MB    26.47s
build/js/frontend-components-inventory.b0938f5f.css            142.91 KB     5.58s
build/js/chrome.css                                            121.74 KB    11.41s
build/js/App.3f6cf847.js                                       106.81 KB    18.22s
build/js/frontend-components-remediations.ec1c6932.js           102.2 KB    15.39s
build/js/App.5da2d433.css                                       40.92 KB     6.28s
build/js/inventoryStyles.e68b22c8.css                            27.6 KB     6.74s
build/js/frontend-components-remediations.7be3c16c.css            6.7 KB     2.53s
build/js/inventoryStyles.dee582d6.js                              1.4 KB     8.20s
build/js/remediationsStyles.83d6d9d2.js                          1.35 KB     3.86s
build/js/remediationsStyles.a24650e6.css                         1.22 KB     2.16s

```

cc @karelhala @ryelo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,7 +114,7 @@
     "@babel/polyfill": "^7.8.7",
     "@babel/runtime": "^7.9.6",
     "@patternfly/patternfly": "^2.71.5",
-    "@patternfly/react-core": "^3.153.13",
```
Comment (karelhala) [inline package.json:116]: We have to use at least version `3.153.7` due to this bug fix https://github.com/patternfly/patternfly-react/pull/3990 I think we can be fine with the extra bundle size for now.

Comment (Hyperkid123) [reply package.json:116]: :cry: ok I will put It back. Hopefully they will fix it soon.

Comment (Hyperkid123) [reply package.json:116]: Ok i used the latest version of PF. That added around 0.5MB to the inventory frontend

Comment (karelhala) [reply package.json:116]: If it's in inventory frontend then it's no big deal.

**Pull request merged**